app/controllers/roles_controller.rb

```diff
@@ -0,0 +1,59 @@
+class RolesController < ApplicationController
+  before_filter :authenticate_user!
+  load_and_authorize_resource
+
+  def index
+    @roles = Role.all
+  end
+
+  def show
+    if @role.users.length == 0
+      @assosciated_users = "None"
+    else
+      @assosciated_users = @role.users.map(&:name).join(", ")
```

Comment:
It's task for view to check for nil and render 'None' or join usernames. Here you should only assign value.